content/docs/visualization/consoles.md

```diff
@@ -9,13 +9,16 @@ The Console templates allow for creation of arbitrary consoles using the [Go
 templating langauge](http://golang.org/pkg/text/template/). These are served
```

Comment:
I think I've never reviewed this doc initially. Must have overlooked it somehow. So I'll also point out some existing stuff... like:

s/langauge/language/

Also, in the first sentence, I'd lose the initial "The".